client/layout/index.jsx

```diff
@@ -184,13 +185,13 @@ Layout = React.createClass( {
 
 export default connect(
 	( state ) => {
-		const { isLoading, section, guidesTour } = state.ui;
+		const { isLoading, section } = state.ui;
 		return {
 			isLoading,
 			isSupportUser: state.support.isSupportUser,
 			section,
 			isOffline: isOffline( state ),
-			shouldShowGuidesTour: guidesTour.shouldShow,
+			guidesState: getGuidesTourState( state ),
```

Comment:
[borderline nitpicky] Other GT-connected pieces use the name `tourState`. Not saying we need to change it to that, but we should be aware that we have different names for the same data lying around.